resources/styles/main.less

```diff
@@ -6,58 +6,82 @@
 @hover-border-width       : 3px;
 
 html, body {
-    margin     : 0;
-    height     : 100%;
-    overflow   : hidden;
-    visibility : hidden;
+  margin: 0;
+  height: 100%;
+  overflow: hidden;
+  visibility: hidden;
 }
 
 /* ---- Editor  ------ */
 .editorContainer {
-    height : calc(~"100vh -" @controls-height);
+  height: calc(~"100vh -" @controls-height);
 }
 
 .CodeMirror {
-    height : 100%;
+  height: 100%;
 }
 
 /* ---- Bottom Menu ----- */
 .border-bottom {
-    height          : @controls-height;
-    //padding         : @controls-padding;
-    background      : @editor-bg-color-darkened;
-    border-top      : 1px solid @editor-bg-color-lightened;
-    color           : white;
-    margin-top      : 0px;
+  height: @controls-height;
+  //padding         : @controls-padding;
+  background: @editor-bg-color-darkened;
+  border-top: 1px solid @editor-bg-color-lightened;
+  color: white;
+  margin-top: 0px;
 }
 
 .menu-items {
-    margin-top : 0px;
-    float      : left;
-    height     : 100%;
+  margin-top: 0px;
+  float: left;
+  height: 100%;
 }
+
 .menu-items li {
-    color       : #d5d5d5;
-    float       : left;
-    margin-left : 15px;
-    font-size   : 12pt;
-    height      : calc(@controls-height - @hover-border-width);
-    padding-top : 12px;
+  color: #d5d5d5;
+  float: left;
+  margin-left: 15px;
+  font-size: 12pt;
+  height: calc(@controls-height - @hover-border-width);
+  padding-top: 12px;
+}
+
+.help-control {
+  padding-top: 12px;
+  margin-right: 15px;
+  float: right;
+  height: calc(@controls-height - @hover-border-width);
+  @media all and (max-width: 465px) {
+    display: none;
+  }
 }
 
 .menu-items li:hover {
-    cursor             : pointer;
-    -webkit-box-shadow : 0px 3px 0px #E64002 inset;
-    -moz-box-shadow    : 0px 3px 0px #E64002 inset;
-    box-shadow         : 0px 3px 0px #E64002 inset;
```

Comment:
Are you using some kind of auto spacing here?, Generally with CSS I like to line it all up, just to make it a bit easier to read, but we don't have to.
I have just realised we have ESLint configuration file but it is not running as part of Gulp. We should get that running by default every time we push, and in the develop task.